cmd/query/app/server.go

```diff
@@ -346,6 +350,7 @@ func (s *Server) Start() error {
 
 // Close stops http, GRPC servers and closes the port listener.
 func (s *Server) Close() error {
+	s.logger.Info("Attempting to close server")
```

Comment:
considering the original error, this log is useless, because the race condition was between closing grpc/gttp/cmux servers, so I would log when those are closed (if possible from their own Close methods if we implement then, but they come from 3rd party library then the best we can do is log just before calling each Close)